package.json

```diff
@@ -69,6 +69,11 @@
           "type": "number",
           "default": 24,
           "description": "The maximum title length before being abbreviated. Set to 0 or less to disable."
+        },
+        "markdown-links.wikilinkAliasDivider": {
```

Comment:
I would rather have `wikiLink` with capital `L` here.

Reply:
I also updated the functions to reflect this.